src/Ocelot/Configuration/File/FileSecurityOptionsExtensions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Ocelot.Configuration.File
+{
+    internal static class FileSecurityOptionsExtensions
```

Comment:
Extensions are required when used types from external libs! But you are able to develop, common!
The method should be moved to extended type!

Reply:
Yes, that is right!